systemcontractindex/stakingindex/mute_handler.go

```diff
@@ -39,15 +39,20 @@ func (eh *eventMuteHandler) HandleStakedEvent(event *abiutil.EventParam) error {
 	if !ok {
 		return errors.Errorf("no owner for token id %d", tokenIDParam.Uint64())
 	}
+	createdAt := eh.block.Height()
+	if eh.timestamped {
+		createdAt = uint64(eh.block.Timestamp().Unix())
```

Comment:
this is to handle v2 bucket after hard-fork right? so no need to check `timestamped`?

Reply:
This can also apply to v3, as MuteOption and TimestampOption are independent.